.gitignore

```diff
@@ -129,5 +129,6 @@ assets/bundles/
 node_modules/
 webpack-stats.json
 staticfiles/
-
+emails/templates/*.html
```

Comment:
Why are we ignoring HTML pages?

Reply:
I don't know, it would be better to ask Justin. I only know how to use the methods Justin gave me.